full-node/db/sov-db/Cargo.toml

```diff
@@ -17,30 +17,44 @@ resolver = "2"
 # Maintained by sovereign labs
 jmt = { workspace = true }
 sov-schema-db = { path = "../sov-schema-db", version = "0.3" }
-sov-rollup-interface = { path = "../../../rollup-interface", version = "0.3", features = ["native", "mocks"] }
+sov-rollup-interface = { path = "../../../rollup-interface", version = "0.3", features = ["native", "mocks", "tokio"] }
 
 # External
 anyhow = { workspace = true }
 arbitrary = { workspace = true, optional = true }
+bincode = { workspace = true, optional = true }
 byteorder = { workspace = true }
 borsh = { workspace = true }
 proptest = { workspace = true, optional = true }
 proptest-derive = { workspace = true, optional = true }
-serde = { workspace = true, features = ["derive"] }
+rocksdb = { workspace = true, optional = true }
+serde = { workspace = true }
 tempfile = { workspace = true, optional = true }
-rocksdb = { workspace = true }
-bincode = { workspace = true }
-tokio = { workspace = true }
+tokio = { workspace = true, optional = true }
 
 
 [dev-dependencies]
 tempfile = { workspace = true }
 
 
 [features]
+default = ["std", "tokio"]
```

Comment:
Why do we need `std` for `sov-db` and `sov-schema-db`? They seem to be almost completely useless without it. Are the `no_std` variants of these crates used/expected to be used anywhere?

Reply:
The rationale behind that is to make as many type representations available to `no-std` as possible, having their concrete implementation as added feature.

The trade-off is that we increase code bloat - especially feature shielding. Wdyt?

Reply:
I think it's a noble intent, but I don't see any possible usage of `sov-db` and `sov-schema-db` in a `no_std` environment at the moment. I may be wrong – if we can think of anything worthwhile, or if my assumption is incorrect, let's do it. As long as `sov_schema_db` and `sov_db` are only ever used in native execution environments with full access to `std`, I don't see a need for `no_std`.